package.json

```diff
@@ -112,7 +112,7 @@
   "dependencies": {
     "@binary-com/binary-document-uploader": "^2.4.6",
     "@binary-com/binary-style": "^0.2.25",
-    "@binary-com/webtrader-charts": "0.5.19",
+    "@binary-com/webtrader-charts": "0.5.20",
```

Comment:
Can we just allow this automatically use the latest version or is there any problem with that?

Reply:
```suggestion
    "@binary-com/webtrader-charts": "^0.5.20",
```